redi-connect-front/src/components/organisms/Carousel.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Section, Container, Element, Columns, Content } from 'react-bulma-components'
+import { useTranslation } from 'react-i18next'
+import Slider from 'react-slick'
+import classnames from 'classnames'
+import Heading from '../atoms/Heading'
+import dragos from '../../assets/images/profile-dragos.svg'
+import khaled from '../../assets/images/profile-khaled.svg'
+import halil from '../../assets/images/profile-halil.svg'
+import './Carousel.scss'
+
+interface Props {
+  headline: string
+  title: string
+  border: 'blue' | 'orange'
+}
+
+const Carousel = ({ headline, title, border }: Props) => {
+  const { t } = useTranslation()
+
+  const quotes = [
+    {
+      title: t('loggedOutArea.homePage.carousel.quotes.quoteKhaled.title'),
+      text: t('loggedOutArea.homePage.carousel.quotes.quoteKhaled.text'),
+      img: khaled
+    },
+    {
+      title: t('loggedOutArea.homePage.carousel.quotes.quoteDragos.title'),
+      text: t('loggedOutArea.homePage.carousel.quotes.quoteDragos.text'),
+      img: dragos
+    },
+    {
+      title: t('loggedOutArea.homePage.carousel.quotes.quoteHalil.title'),
+      text: t('loggedOutArea.homePage.carousel.quotes.quoteHalil.text'),
+      img: halil
+    }
+  ]
+
+  const settings = {
+    dots: true,
+    infinite: true,
+    arrows: false,
+    speed: 1000,
+    autoplaySpeed: 10000,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    fade: true,
+    autoplay: true,
+    pauseOnHover: true,
+  };
+
+  return (
+    <Section className="default-background">
+      <Container>
```

Comment:
What is the reason behind keeping both of the container?

Reply:
deleted